modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -4,11 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, metadata, dataweave, data weave, datawave, datamapper, dwl, dfl, dw, payload, attribute, variable, flow, error, message
 
-DataWeave can select data from objects, arrays, and variables that store them,
-both
-xref:dataweave-variables-context.adoc[predefined Mule Runtime variables]
-and in xref:dataweave-variables.adoc[DataWeave variables]. You can use selectors
-in Mule modules, connectors, and components that accept DataWeave expressions.
+DataWeave can select data from objects, arrays, and variables that store them.
+You can use selectors in Mule modules, connectors, and components that
```

Comment:
The "that store them" part of the first sentence is not clear. Can you remove this or word this in a different way? What does "them" refer to